airbyte-integrations/connectors/source-freshcaller/Dockerfile

```
@@ -35,5 +35,5 @@ COPY source_freshcaller ./source_freshcaller
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.1.0
+LABEL io.airbyte.version=0.1.1
```

```suggestion
LABEL io.airbyte.version=0.2.0
```
I'm suggesting minor version bump as you're adding new field: could be considered it's a new feature.

---

agreed, updated